src/test/java/com/ericsson/eiffelcommons/subscriptionobjecttest/SubscriptionObjectTest.java

```diff
@@ -79,11 +78,31 @@ public void testSetRestPostBodyMediaType() {
         restPostSubscription.setRestPostBodyMediaType(MediaType.APPLICATION_FORM_URLENCODED);
 
         String actualRestPostBodyMediaType = restPostSubscription.getSubscriptionJson()
-                                                                 .get("restPostBodyMediaType")
-                                                                 .toString();
+                                                                 .getString("restPostBodyMediaType");
         assertEquals("application/x-www-form-urlencoded", actualRestPostBodyMediaType);
     }
 
+    @Test
+    public void testSetNotificationBody() {
+        String body = "Body";
+        mailSubscription.addNotificationBody(body);
+
+        String actualNotificationBody = mailSubscription.getSubscriptionJson()
+                                                        .get("notificationMessageKeyValues")
```

Comment:
use getString as in the other places.

Reply:
notificationMessageKeyValues is not a string though, it's an object.